APMrover2/Rover.h

```diff
@@ -368,6 +368,7 @@ class Rover : public AP_HAL::HAL::Callbacks {
 
     // last wheel encoder update times
     float wheel_encoder_last_angle_rad[WHEELENCODER_MAX_INSTANCES];     // distance in radians at time of last update to EKF
+    float wheel_encoder_last_distance_m[4];                             // distance in meters at time of last update to EKF (for reporting to GCS, use length 4 to fit the mavlink message)
```

Comment:
why 4 ? it shouldn't be WHEELENCODER_MAX_INSTANCES ?

Reply:
The corresponding WHEEL_DISTANCE message holds array of size 4. So I use wheel_encoder_last_distance_m[4] to pass it to mavlink_msg_wheel_distance_send() procedure, which needs size 4 array.

Reply:
Probably because we might have up to 4 driven wheels?